src/hooks/imagePicker.ts

```diff
@@ -45,3 +47,40 @@ export const useSelectImage = (
 
   return { imageUri, selectImage };
 };
+
+export const useCaptureImage = (
+  onChange?: <T>(
+    setter: React.Dispatch<React.SetStateAction<T>>
+  ) => React.Dispatch<React.SetStateAction<T>>,
+  allowsEditing?: boolean,
+  aspect?: [number, number],
+  quality?: number,
+  mediaTypes?: MediaTypeOptions
+) => {
+  const [imageUri, setImageUri] = useState<string>();
+
+  const captureImage = useCallback(async () => {
+    const { status } = await requestCameraPermissionsAsync();
+
+    if (status !== PermissionStatus.GRANTED) {
+      Alert.alert(texts.errors.image.title, texts.errors.image.body);
+      return;
+    }
+
+    // this allows for proper selecting and cropping to 1:1 images (and not videos)
+    // for more details about options see: https://docs.expo.dev/versions/latest/sdk/imagepicker/#imagepickermediatypeoptions
+    const result = await launchCameraAsync({
+      mediaTypes: mediaTypes ?? MediaTypeOptions.Images,
+      allowsEditing: allowsEditing ?? true,
+      aspect: aspect ?? [1, 1],
```

Comment:
what about the fallback of 1, 1? can we please keep the original foto aspect ratio?

Reply:
If we remove the `aspect`, we have the original aspect ratio. But [1, 1] is also available for photos selected from the gallery. Should we update this too?

Reply:
both should be identical, whether original format or 1:1

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1007/commits/f9c2849385f73cb7ac9388da8f85a0675817a9aa